commands/chain_daemon_test.go

```diff
@@ -127,3 +134,71 @@ func TestChainLs(t *testing.T) {
 		assert.Contains(t, chainLsResult, `"height":"1"`)
 	})
 }
+
+func TestChainImportExport(t *testing.T) {
+	tf.IntegrationTest(t)
+
+	ctx, env := fastesting.NewTestEnvironment(context.Background(), t, fast.FilecoinOpts{
+		DaemonOpts: []fast.ProcessDaemonOption{fast.POBlockTime(50 * time.Millisecond)},
+	})
+	// Teardown after test ends
+	defer func() {
+		err := env.Teardown(ctx)
+		require.NoError(t, err)
+	}()
+	genesisNode := env.GenesisMiner
+
+	// create a node that will export its chain
+	exportNode := env.RequireNewNodeWithFunds(1000)
+	require.NoError(t, series.Connect(ctx, genesisNode, exportNode))
+
+	// create blocks with no messages
+	series.CtxMiningOnce(ctx)
+
+	// create block with 1 message
+	require.NoError(t, series.SendFilecoinDefaults(ctx, genesisNode, exportNode, 10))
+	series.CtxMiningOnce(ctx)
+
+	// create block with many messages
+	for i := 0; i < 20; i++ {
+		require.NoError(t, series.SendFilecoinDefaults(ctx, genesisNode, exportNode, 10))
+		require.NoError(t, series.SendFilecoinDefaults(ctx, exportNode, genesisNode, 10))
```

Comment:
Will change this to use the SendMessage action instead as these series mine blocks.